app/javascript/src/components/editor/Modals/TranslationKeysModal.svelte

```diff
@@ -44,9 +27,12 @@
         Select languages ({$selectedLanguages.length})
       </button>
 
-      {#if Object.keys($orderedTranslationKeys).length}
-        <h4 class="mb-1/8"><strong>Keys</strong></h4>
+      <h4 class="flex align-center justify-between mb-1/8">
+        <strong>Keys</strong>
+        <button class="button button--primary button--small button--square" on:click={() => { selectedKey = createNewTranslationKey; showLanguageSettings = false }}>Add</button>
```

Comment:
Calling creaetNewTranslationKey like that here will assign the function, not the value, right?
```suggestion
        <button class="button button--primary button--small button--square" on:click={() => { selectedKey = createNewTranslationKey(); showLanguageSettings = false }}>Add</button>
```

Reply:
`createNewTranslationKey` is a symbol defined defined in `utils/translations.ts`

Reply:
Ah of course, it sounds like a function, like it creates something

Reply:
Maybe I should drop the `create`, and add `Symbol` at the end?: `newTranslationKeySymbol`

Reply:
Sure! Just `newTranslationKey` works too I think